src/components/InfoText/index.tsx

```diff
@@ -13,32 +13,27 @@ interface InfoTextProps {
 const InfoText: React.FC<InfoTextProps> = ({ days, month, beginningTime, endTime }) => {
     return (
         <>
-            <HeadingText text="O que é o Fallstack?" />
-            <EventInfos days={days} month={month} beginningTime={beginningTime} endTime={endTime} />
+            <HeadingText text="FALLSTACK" />
```

Comment:
Same here, please ensure that, in light mode, the heading text color changes to black